recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - pip
     - {{ mpi }}
   run:
+    - libgfortran-ng  # [aarch64 and mpi=='mpich']
```

Comment:
run_exports is being excluded in general on aarch64, which is a severe problem and should probably stop all aarch64 builds until it's fixed. I think it's a bug in conda-build, but not sure. Probably all aarch64 builds should stop until this is resolved, because it's going to result in *lots* of broken packages.

Let's make sure to remove this line here, because it's hiding a major problem that really should be preventing this from succeeding.

Reply:
root issue: https://github.com/conda/conda-build/issues/3583

Reply:
But this only affects aarch64? Since the issue mentions ppc64le as well, but seams not to be the case?

Reply:
It affects more (linux-64 as well), it mainly has to do with the conda-build version and when the given build of mpich was uploaded. ppc64 was uploaded earlier, so missed the bug I think.